scss/variables/colors.js

```diff
@@ -24,7 +25,8 @@ chassis.colors = {
 			"base": "#3F51B5",
 			"light": "#5C6BC0",
 			"dark": "#3949AB",
-			"darker": "#303F9F"
+			"darker": "#303F9F",
+			"notification": "#3F51B5"
```

Comment:
You can add a default.grey or a new palette 'grey':{'dark':#333333} Lets discuss in next meeting.

 Anyways if you set your background  variable directly to the color code value rather than the entire map, things will get simpler and allow you more choice, since then you need not access using map-deep-get(, , notifications) everytime.